src/components/LoanCards/LoanCardController.vue

```diff
@@ -13,12 +14,15 @@
 		:is-funded="isFunded"
 		:is-selected-by-another="isSelectedByAnother"
 		:amount-left="amountLeft"
+		:show-category-description="showCategoryDescription"
+		:experiment-data="experimentData"
```

Comment:
@ksekhri  / @mcstover I can imagine one or more experiments and hence the creation of `experimentData` (type: Array). Also, I believe having the `loanChannel` (type: Object) would be useful for any future loan card views.

Reply:
@foochris I think the experimentData array could be useful. You'll want to change how your accessing it though to use find or filter instead of keying off of an index.

As for the loan channel, I don't want to add this to the loan as it will in your case contain the same loan you passed in and in other cases contain all of the loans we queried for a loan channel. If there is a need to pass down the Loan Channel ID, Name or other meta data we can extract that when the time comes.

Reply:
@mcstover If the loan channel object is not to be passed (and used to display the description), what if we had a property for generic text instead ('text', type: String)?

https://github.com/kiva/ui/blob/a688861be1529e03e95f9f67791f4b750f673d3c/src/components/LoansByCategory/FeaturedHeroLoan.vue#L7

Reply:
@mcstover PR updated with your suggestion of moving the loan description to the wrapper.

Reply:
@ksekhri / @mcstover PR updated with `experimentData` as `type: Object`.